### PR TITLE
media-gfx/cura: new ebuild

### PR DESCRIPTION
**Pull request by tomboy-64**
This is Ultimakers application for mesh-slicing and 3d-printing.
~~This commit depends on my PRs for [bumped Protocol Buffers (#1295)](https://github.com/gentoo/gentoo/pull/1295)~~ and ~~[pyserial with enabled python 3.5 (#1297)](https://github.com/gentoo/gentoo/pull/1297)~~.
Please review.

@gentoo/proxy-maint
@gentoo/3dprint
@gentoo/python
@axs-gentoo

### CODE REVIEW

File: media-gfx/cura/cura-2.1.0_beta.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+LICENSE="AGPL-3"
+SLOT="0"
+IUSE="+usb"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-libs/libarcus:=[${PYTHON_USEDEP}]
+	dev-python/uranium[${PYTHON_USEDEP}]
+	usb? ( dev-python/pyserial[${PYTHON_USEDEP}] )
+	~media-gfx/curaengine-${PV}
+	sys-devel/gettext"
```
Comment (gktrk) [inline media-gfx/cura/cura-2.1.0_beta.ebuild]: shouldn't gettext be a build-time dependency?

Comment (tomboy-64) [inline media-gfx/cura/cura-2.1.0_beta.ebuild]: Correct. Thanks. Fixed.

**Comment by monsieurp** [comment]
It looks good to me. Let's get someone from @gentoo/python to look through this PR as well. @floppym maybe?

**Comment by tomboy-64** [comment]
It's fine, but I'm still struggling with opengl issues and python deps.

**Pull request closed without merging**